src/utils.ts

```diff
@@ -93,3 +93,8 @@ export function isWithinRootPath(file: string) {
 
 	return workspace.rootPath != null && file.startsWith(workspace.rootPath + path.sep); 
 }
+
+export function log(message: any): void {
+	if (config.verbose)
```

Comment:
Added #89 to remove `verbose`; also suggested we tweak the logging (that sends warnings to warning dialogs) to use the same "developer" detection as analytics.